src/client/components/StudyGroupLogo/StudyGroupLogo.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './StudyGroupLogo.styles.css';
+
+export const StudyGroupLogo = (props) => {
+  return (
+    <i>
```

Comment:
Unnecessary element `i`.

Reply:
From the wireframes in the figma, it shows that there are two parts in the logo.
1. the round circle ---> it is through <i> tag
2. actual logo ----> it is through <img>tag

Reply:
Got it, but then replace it with either span or div - do not use `i`.

Reply:
yes it is changed to span